docs/section6/Rule6-9.md

```diff
@@ -0,0 +1,32 @@
+
+# Lighting - Rule 6-9
+
+**Rule ID:** 6-9  
+**Rule Description:** Baseline building is modeled with automatic shutoff controls in buildings >5000 sq.ft.  
+**Appendix G Section:** Section G3.1-6 Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:**  None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+  1. Building total area is more than 5,000sq.ft.  
+
+**Manual Check:**  
+
+  1. Where building total area is more than 5,000sq.ft., manual review is required to verify automatic shutoff controls are modeled correctly in the Baseline.  
+
+**Function Call:** None  
+
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic: 
+
+- For each building_segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
+  - For each thermal_block in building segment: ```for thermal_block_b in building_segment.thermal_blocks:```  
+    - For each zone in thermal block: ```for zone_b in thermal_block_b.zones:```  
+      - For each space in zone: ```for space_b in zone.spaces:```  
+        - Add space floor area to building total floor area: ```building_total_area_b += space_b.floor_area```  
+- **Applicability Check 1:**```if building_total_area_b > 5000:```  
+
+- **Manual Check 1:** Manual review is required where building total area is more than 5,000sq.ft. to verify automatic shutoff controls are modeled correctly in the Baseline.  
```

Comment:
Since automatic lighting controls are also required in the proposed design, I think this rule has to confirm that during unoccupied hours the same lighting schedules are modeled in B-RMR and P-RMR. In addition, The hourly fractions used in these schedules should be 0 or a low number because only the emergency lighting would stay on. I'll discuss it with others and copy you.

from 90.1 2004:
9.4.1.1 Automatic Lighting Shutoff. Interior lighting
in buildings larger than 5000 ft2 shall be controlled with an
automatic control device to shut off building lighting in all
spaces.

Reply:
@mkarpman and @xingcx - I agree with this assessment. The RDS would review the occupancy schedule and the lighting schedule. for times where occupancy is <=10%, the lighting schedule should be very low, only accounting for emergency lighting. This RDS should verify 2 things (1) lighting schedule is very low (0 or close to 0) when occupancy schedule is <=10% in the B-RMR (2) P-RMR = B-RMR during unoccupied hrs (when occupancy is <=10%)

Reply:
I think the trigger should be the schedule in building.building_open_schedule, not space.occupant_multiplier_schedule, because based on the quote from 90.1 2004 above the rule is for when the entire building is unoccupied. Agreed?

Reply:
@mkarpman - we couldn't hear you on the call. The approach you have summarized here looks good. we can use the building.building_open_schedule to identify unoccupied times (this would be when schedule =0)
during those times, we will verify that the lighting scheduled in B=P for each space.
This would need to use the lighting function which calculates a W/ft2 weighted average lighting schedule for each space.